web/marketing/pages/index.html

```diff
@@ -4,7 +4,7 @@
 {{define "head"}}
     <title>Referral Admin Portal</title>
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
-    <link rel="stylesheet" type="text/css" href="/static/css/style.css">
+    <link rel="stylesheet" type="text/css" href="static/css/style.css">
```

Comment:
Let's add a `{{.BaseURL}}` which you can supply via a config. I think it still should use absolute URL-s because otherwise it's easy to mess up paths. e.g. `/hello` works, but `/hello/world` doesn't.

Reply:
I support that idea!

Reply:
You can supply the base url via template funcs. e.g. register a func called `Site`, which returns a type that contains field `BaseURL`, then you can use it as: `{{ Site.BaseURL }}`